src/20_Components/amp-experiment.html

```diff
@@ -39,7 +38,7 @@
 </head>
 <body>
 
-  <!-- ## Basic Usage -->
+  <!-- ## Basic usage -->
```

Comment:
Can you also fix title case for `Development Tip`

Reply:
done